OrbitQt/CallTreeWidget.cpp

```diff
@@ -269,33 +269,38 @@ void CallTreeWidget::onCustomContextMenuRequested(const QPoint& point) {
   bool enable_copy = ui_->callTreeTreeView->selectionModel()->hasSelection();
 
   QMenu menu{ui_->callTreeTreeView};
-  if (enable_expand_recursively) {
-    menu.addAction(kActionExpandRecursively);
-  }
-  if (enable_collapse_recursively) {
-    menu.addAction(kActionCollapseRecursively);
-    menu.addAction(kActionCollapseChildrenRecursively);
-  }
+
+  menu.addAction(kActionExpandRecursively);
```

Comment:
Not sure if this (and everything below) would be more readable as a one-liner:
`menu.addAction(kActionExpandRecursively)->setEnabled(enable_expand_recursively);`

It's a nitpick though, I can live with the current option as well.

Reply:
Aah that's how you do it, I had completely missed that `addAction` returns the `QAction`.